rtv/utils.py

```diff
@@ -16,6 +16,27 @@
 
 ESCAPE = 27
 
```

Comment:
We need to consider combining this with the one in `main.epilog`. I actually like this version more, it's very succinct.

I'm thinking something like this in main.py
```python
from rtv.utils import HELP

epilog = """blah blah blah"""
epilog += HELP
```